### PR TITLE
Task/aj 165 create task with all fields

### PR DESCRIPTION
**Pull request by GustavoZHA**

### CODE REVIEW

File: src/main/java/core/utils/JsonFileManager.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ *
+ * @author Gustavo Huanca
+ */
+
+package core.utils;
+
+import io.cucumber.messages.internal.com.google.gson.JsonObject;
+import io.cucumber.messages.internal.com.google.gson.JsonParser;
+import org.json.JSONObject;
+
```
Comment (JorgeRCV) [inline src/main/java/core/utils/JsonFileManager.java]: Delete space

File: src/main/java/core/utils/JsonFileManager.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Map;
+
+public class JsonFileManager {
+
+    /**
+     * Reads a file json.
```
Comment (JorgeRCV) [inline src/main/java/core/utils/JsonFileManager.java]: Reads a json file sounds better

File: src/test/java/cucumber/steps/ApiSteps.java [hunk 3]
```diff
@@ -20,11 +20,14 @@
...
 import core.api.ApiRequestBuilder;
 import core.api.ApiResponse;
+import core.utils.JsonFileManager;
+import core.utils.MapStringStringToStringObject;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
-import org.json.JSONObject;
 import org.testng.asserts.SoftAssert;
+
```
Comment (JorgeRCV) [inline src/test/java/cucumber/steps/ApiSteps.java]: Delete space

**Review by Raymundogs — APPROVED** [review]

File: src/test/resources/features/tasks/create_task.feature [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      | priority        | 3                                                           |
+      | due_date        | 1508369194377                                               |
+      | due_date_time   | false                                                       |
+      | time_estimate   | 8640000                                                     |
+      | start_date      | 1567780450202                                               |
+      | start_date_time | false                                                       |
+      | notify_all      | true                                                        |
+      | parent          | null                                                        |
+      | links_to        | null                                                        |
+      | Custom fields   | src\test\resources\json\tasks\custom_fields_by_defects.json |
```
Comment (JorgeRCV) [inline src/test/resources/features/tasks/create_task.feature:21]: Is there another way to manage this path, have a path hardcoded makes the scenario hard to read

Comment (GustavoZHA) [reply src/test/resources/features/tasks/create_task.feature:21]: The main reason the user or whatever wants to see the contents of custom_fields_by_defects.json is to know where it is.

**Review by JorgeRCV — APPROVED** [review]

**Pull request merged**